spe_acceptance.py

```diff
@@ -189,11 +201,14 @@ def write_to_file(filename, LED_multihist, noise_multihist):
     data['LED_charge'] = list(np.array(LED_charges).T)
     data['noise_amplitude'] = list(np.array(noise_amplitudes).T)
     data['noise_charge'] = list(np.array(noise_charges).T)
-
     filename = os.path.join(data_dir_base, filename)
-    data.to_hdf(filename, key='data')
-    print("Data written to %s" % filename)
-
+    #give the directory and the file the right group permissions
+    if request.user.groups.filter(name__in=['pi-lgrandi']).exists():
```

Comment:
Have these changes been checked? I'm pretty sure this line would fail. `request` is defined nowhere in the script. 

Also what is this line supposed to be doing? 

We also need to do a check that the file is owned by the user. Otherwise the chown and chmod commands will fail when e.g. Andy first processes the data and Evan processes it after the file has already been created (file owned by Andy, trying to change permissions as Evan)